tests/Resources/Page/OCPDashboard/Pods/Pods.robot

```diff
@@ -0,0 +1,17 @@
+*** Settings ***
+Resource   ../../OCPDashboard/Page.robot
+Resource   ../../ODH/ODHDashboard/ODHDashboard.robot
+Library         ../../../../libs/Helpers.py
+
+*** Keywords ***
+Get Pod Logs
+  [Arguments]  ${namespace}  ${pod_search_term}
+  Navigate To Page    Workloads    Pods
+  Search Last Item Instance By Title in OpenShift Table  search_term=${pod_search_term}  namespace=${namespace}
```

Comment:
it always picks the most recent one. It is one of those keyword that may just work for Anaconda, where you need to take the latest pod every time. However, I think this keyword will be soon deprecated by the new one in the OpenshiftCLI library. 
To avoid confusion, I will take this back in the Anaconda test suite. Is that ok?

Reply:
I can imagine cases where we might want to get the pod logs for multiple pods, but I guess in that case we can just call this keyword multiple times. I'd leave this here until we have a replacement in the openshiftcli library, and then we can eventually remove it.